custom_components/tuya_local/devices/airx_h8_humidifier.yaml

```diff
@@ -0,0 +1,119 @@
+name: Humidifier
+products:
+  - id: orypuo7ksduqze3x
+    manufacturer: airx
+    model: H8Pro
+entities:
+  - entity: humidifier
+    class: humidifier
+    dps:
+      - id: 1
+        name: switch
+        type: boolean
+        mapping:
+          - dps_val: true
+            icon: "mdi:air-humidifier"
+          - dps_val: false
+            icon: "mdi:air-humidifier-off"
+      - id: 13
+        name: humidity
+        type: integer
+        range:
+          min: 30
+          max: 80
+      - id: 14
+        name: current_humidity
+        type: integer
+      - id: 24
+        type: string
+        name: mode
+        mapping:
+          - dps_val: AUTO
+            value: auto
+          - dps_val: LOW
+            value: eco
+          - dps_val: MIDDLE
+            value: normal
+          - dps_val: HIGH
+            value: boost
+          - dps_val: SLEEP
+            value: sleep
+  - entity: binary_sensor
+    category: diagnostic
+    translation_key: machine_separation_status
```

Comment:
I think this can be class opening, or tamper, rather than defining a new translation key. My interpretation is that this is telling that you have dismantled the device to replace the filter etc, is that correct? Tamper is more for alarm systems where dimantling is an alarm condition, and also does not have good translations for the on/off states, so opening may be better.